acvm-repo/acir_field/src/field_element.rs

```diff
@@ -405,6 +409,24 @@ mod tests {
         assert_eq!(max_num_bits_bn254, 254);
     }
 
+    #[test]
+    fn test_endianness() {
+        let field = FieldElement::<ark_bn254::Fr>::from(0x1234_5678_u32);
+        let le_bytes = field.to_le_bytes();
+        let be_bytes = field.to_be_bytes();
+        
+        // Check that the bytes are reversed between BE and LE
+        let mut reversed_le = le_bytes.clone();
+        reversed_le.reverse();
+        assert_eq!(be_bytes, reversed_le);
+
+        // Verify we can reconstruct the same field element from either byte order
+        let from_le = FieldElement::from_be_bytes_reduce(&reversed_le);
+        let from_be = FieldElement::from_be_bytes_reduce(&be_bytes);
+        assert_eq!(from_le, from_be);
```

Comment:
There already is a test to check that these are the same bytes, so this should trivially pass. 

Perhaps a more interesting case would be to add a `from_le_bytes_reduce` method.